bin/selenium-standalone

```diff
@@ -53,7 +53,11 @@ function parseCommandAndOptions(javaPath) {
   // Merge default options, options from config file then command line options
   options = merge({}, defaultConfig, configFromFile, options);
 
-  options.seleniumArgs = seleniumArgs;
+  if (seleniumArgs.length) {
```

Comment:
@vvo : That would not respect the "command line takes precedence over config file", no ?